src/content/en/2020/compression.md

```diff
@@ -235,13 +242,15 @@ Additionally, the percentage of brotli compression is higher for third-party con
   <figcaption>{{ figure_link(caption="First-party versus third-party compression by device type.", sheets_gid="862864630", sql_file="19_03.party_of_content_encoding.sql.sql") }}</figcaption>
 </figure>
 
-Comparing with [last year's results](https://almanac.httparchive.org/en/2019/compression#first-party-vs-third-party-compression), we can see that there was a significant increase in the use of compression, notably brotli for first parties, almost to the point that the use of compression is around 40% for both first and third party and for desktop and mobile. However within the requests that do use compression, for first party the ratio of brotli compression is only 18%, while the ratio for third party is 27%.
+When we compare compression techniques between first and third parties, we can see that third-party content tends to be compressed more than first-party content. Additionally, the percentage of brotli compression is higher for third-party content. This is likely due to the number of resources served from the larger third parties that typically support brotli, such as Google and Facebook.
+
+{# TODO(authors/analysts): Would be nice to have a year-over-year comparison chart here. #}
```

Comment:
@mo271 @AbbyTsai 

This isn't critical, just a missed opportunity to have a yoy comparison chart.

Reply:
We decided against adding yoy comparison charts, since the results of it were not super surprising.